app/services/moderator/banisher.rb

```diff
@@ -47,7 +49,9 @@ def delete_comments
 
       user.comments.find_each do |comment|
         comment.reactions.delete_all
-        CacheBuster.new.bust_comment(comment.commentable, user.username)
+        if !comment.commentable.nil?
```

Comment:
This can be accomplished with fewer codes.
```suggestion
        if comment.commentable
```